### DISCUSSION TITLE
Could this be implemented

### DISCUSSION THREAD

**Discussion by ghost** [chris-greening/instascrape#74] [General]
Im not sure if this is already a  feature as im new, but I found this repo when googling how to extract the likes from an instagram post using python. Is there a way of implementing something that will allow  a user to extract a post's attributes e.g. likes, views, comments, caption, hashtags, tags ... ?

e.g. 
postName.likes returns the number of likes of the post
postName.comments returns the number of comments of the post
postName.caption returns the caption of the post'

EDIT: NEVER MIND, I FOUND A WAY TO DO THIS BUT I DON'T KNOW HOW TO DELET THIS DISCUSSION

**Comment by chris-greening** [comment]
lol I'm glad you were able to find what you needed! Thanks for checking out the lib 😃

**Reply by daniqueosinga** [reply]
How did you do it?
